php-checks/src/test/resources/checks/CountInsteadOfEmptyCheck.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+if (count($a) >= 0) { // Noncompliant
+//  ^^^^^^^^^
+  echo $a[0];
+}
+
+if (count($a_2) >= 0) { // Compliant
+  echo $a_2;
+}
+
+if (0 < count($a_3)) { // Noncompliant
+  echo $a_3[0];
+}
+
+if (count($a_4) >= 1) { // Compliant
+  echo $a_4[1];
+}
+
+$b = getData();
+echo $b[0];
+if (count($b) > 0) { // Noncompliant
+  echo "foo";
+}
+
+if (count($c) >= 0) { // Compliant - We are not sure that $b is an array
```

Comment:
The variable name in the comment is not correct.

Reply:
Fixed.